tf2onnx/onnx_opset/tensor.py

```diff
@@ -1248,3 +1248,156 @@ def version_10(cls, ctx, node, **kwargs):
         target_dtype = TensorProto.INT64
         if seq_len_dtype != target_dtype:
             ctx.insert_new_node_on_input(node, "Cast", node.input[1], to=target_dtype)
+
+
+@tf_op("ReverseV2")
+class ReverseV2:
+    @classmethod
+    def version_10(cls, ctx, node, **kwargs):
+        # T output = ReverseV2(T input, int32|int64 seq_lengths, @int seq_dim, @int batch_dim)
+        # 
+        # Implement tensorflow ReverseV2 op using multiple ReverseSequence (for each axis)
+        # and Transpose ops. We sort the axis vector at the start. Each axis can be reversed only
+        # once (in tf) and so we can compute the transpose for each axis (other than 0), feed the
+        # tensor to a ReverseSequence node and finally transpose again to get back the original shape. 
```

Comment:
hm.. seem you forgot to sort axes.

Reply:
what I meant is you need to sort the axes in order to handle the 0th axis first.

Reply:
@fuzzeBatman kindly ping.

Reply:
Ah! Sorry. I missed this notification.

Reply:
@lucienwang1009 Sorting does not affect the result. Is this for using `np.argsort` ?

Reply:
Sorting will make us deal with the axis 0 first. see line 1326 comment.

Reply:
I lost track of my own code. Thanks

Reply:
Pushed the changes. Runs sort on the axis vector.